eve/workers/openstack-multiple-nodes/terraform/scripts/bootstrap-config.sh

```diff
@@ -21,7 +21,7 @@ networks:
 ca:
   minion: $(cat /etc/salt/minion_id)
 archives:
-  - /var/tmp/metalk8s
+  - /home/centos/metalk8s.iso
```

Comment:
Minor: Can you just remove this change so that I do not have conflict in my snapshot PR :) (We cannot use this path easily as it differ from RHEL and CentOS. But we could change it a bit later to use iso

Reply:
Right... I can wait for your PR to merge and handle conflict, because I'd really like to have one pre-merge test checking that this behaviour works.

Reply:
Rebased and updated my commit to make sure we test using an ISO in BootstrapConfig (`/archives/metalk8s.iso`), mounted at a non-standard path (`/var/tmp/metalk8s`) - cc @TeddyAndrieux.